diff --git a/node_modules/@types/rdfjs__express-handler/index.d.ts b/node_modules/@types/rdfjs__express-handler/index.d.ts
index 7607bf2..6879b39 100644
--- a/node_modules/@types/rdfjs__express-handler/index.d.ts
+++ b/node_modules/@types/rdfjs__express-handler/index.d.ts
@@ -28,6 +28,7 @@ interface RdfHandlerOptions {
     formats?: Formats | undefined;
     defaultMediaType?: string | undefined;
     baseIriFromRequest?: boolean | BaseIriFromRequest | undefined;
+    sendTriples?: boolean | undefined;
 }
 
 interface RdfHandler {
